### ISSUE TITLE
Ability to disable copying copyright files

### ISSUE THREAD

**Issue by TheAssassin** [probonopd/linuxdeployqt#194]
The copyright file copying feature is slowing down calls to linuxdeployqt, has a heavy impact on the disk IO, and I consider it more a "nice to have" than really required.

Therefore, I'd like to have a possibility to deactivate this feature, e.g., using a CLI parameter. This is primarily meant to improve my local development workflow, where bundling copyright files is not necessary, and probably a waste of valuable time.

**Comment by probonopd** [comment]
The performance has to do with the fact that we are doing things recursively, redundantly... some restructuring of the code is needed. If we would work on each file only once, the overhead from the license files should be negligible.

**Comment by TheAssassin** [comment]
I just figured that my method to analyze the performance issue might affect my thoughts. I guess the dpkg-query processes are not terminated right after they finished generating output (i.e., `pclose()` might be called too late). We should look into that.

**Comment by probonopd** [comment]
Is this wrong?

https://github.com/probonopd/linuxdeployqt/blob/e129d347bf0d26ba9fa49d616c9da7ef3ccc41f3/tools/linuxdeployqt/shared.cpp#L306-L308

**Comment by TheAssassin** [comment]
Yes, of course, see https://stackoverflow.com/a/25063904. Without this, I suspect that as soon as the QProcess object is deleted, `close()` is terminated implicitly.
